### ISSUE TITLE
Cannot read property 'setRootController' of undefined

### ISSUE THREAD

**Issue by gwendall** [wix/react-native-navigation#77]
I am having this error when I start the app. Any idea what I am missing ?

**Comment by gwendall** [comment]
My bad I forgot the [Xcode config part](https://github.com/wix/react-native-navigation#installation---ios). Closing.

**Comment by cesargdm** [comment]
I've keep getting this error after following all the instructions for iOS, linking, changing AppDelegate.m, removing node_modules, adding again.
In the remote Debugger I first get the warning `Native component for "RCCToolBar" does not exist` after that I get error `Cannot read property 'setRootController' of undefined`

**Comment by gokulkulkarni1987** [comment]
Hi all, even I am also facing the same issue as mentioned by @cesargdm .

**Comment by theoutlander** [comment]
I solved the issue by following the instructions listed @ https://wix.github.io/react-native-navigation/#/installation-ios. I happened to skip that step as I got confused with a screenshot link on that page that took me to https://facebook.github.io/react-native/docs/linking-libraries-ios.html#step-1 (I thought this was a command line alternative and wasted 2 days wondering why only the example project worked!).

My React-Native version is 0.55.4.
